src/Components/EnvironmentPicker/EnvironmentPickerManager.ts

```diff
@@ -13,10 +13,15 @@ import {
 export const getEnvironmentDisplayText = (env: string | IAzureResource) => {
```

Comment:
![image](https://user-images.githubusercontent.com/57726991/186237482-67cb6e00-f481-45b9-adc6-9a8e4aeff2fd.png)

Reply:
had commented and marked as "resolved" this on the original PR